soc/arm/nordic_nrf/nrf51/Kconfig.series

```diff
@@ -12,7 +12,6 @@ config  SOC_SERIES_NRF51X
 	select SOC_FAMILY_NRF
 	select NRF_RTC_TIMER
 	select CLOCK_CONTROL
-	select CLOCK_CONTROL_NRF
```

Comment:
For consistency it would be then nice to also update the nrf52_bsim Kconfig.board

Reply:
Updated